src/audio/kpb.c

```diff
@@ -794,12 +794,11 @@ static uint64_t kpb_draining_task(void *arg)
 	size_t size_to_copy;
 	bool move_buffer = false;
 	uint32_t drained = 0;
-	uint64_t time_start;
-	uint64_t time_end;
+	uint64_t timer;
```

Comment:
It's not a timer, ticks or time may be better names.

Reply:
updated